index.js

```diff
@@ -921,7 +956,7 @@ export function load_placements(force_load = false) {
   let elements = Array.prototype.slice.call(node_list);
 
   if (elements.length === 0) {
-    console.debug("No ad placements found.");
+    logger.warn("No ad placements found.");
```

Comment:
Should we provide more context here? I feel like as a user this message is not super helpful..

```suggestion
    logger.warn("No ad placements found. Check your publisher account is correct and has ads enabled.");
```